commander/src/utils/cryptography.ts

```diff
@@ -94,7 +94,9 @@ export const { getKeys } = cryptography;
 export const getAddressFromPublicKey = (
 	publicKey: string,
 ): { readonly address: string } => ({
-	address: cryptography.getAddressFromPublicKey(publicKey),
+	address: cryptography
+		.getAddressFromPublicKey(Buffer.from(publicKey, 'hex'))
+		.toString('hex'),
```

Comment:
Why to return `string` for address? This function can be redefined to use buffer as input/output.

Reply:
Same as above, this usage needs to be removed when migrating to core